openedx/core/djangoapps/discussions/docs/decisions/0005-in-context-discussion-course-blocks.rst

```diff
@@ -0,0 +1,85 @@
+Making Linked In-Context Discussions Available to MFEs
+======================================================
+
+
+Status
+------
+
+Proposal
+
+
+Context
+-------
+
+In the `ADR 0004 <./0004-in-context-discussions-linking.rst>`_ we investigated
+how to link a discussion to a unit. We also need some way to make these linked
+discussions available to the frontend so they can display those in-context
+discussions in the correct view.
+
+
+Requirements
+------------
+
+An API to access linked discussions for a Unit.
+
+
+Decision
+--------
+
+A direct link to the topic that needs to be embedded can be generated
```

Comment:
You said that "a direct link to the topic" will be provided by edx-platform using the course blocks API, but when you access the API using `requested_fields=discussions_embed_url`, it returns an object with different kinds of URLs. 

Can specify which is the embeddable URL among those when you call it "a direct link"?

Reply:
This is just the standard default response of the [course blocks API](https://github.com/edx/edx-platform/blob/d701b423f123a484d059d211fe9de927f72e42fb/lms/djangoapps/course_api/blocks/views.py#L26). It returns some standard fields, and other fields can be requested in addition as needed. 

In this case, the idea is to add a new field option as specified above and have that add a new return value called "discussions_embed_url". I've cleaned up some the other URLs so it's easier to focus on this new field.

Reply:
Ok, taking a look.